gen/internals.py

```diff
@@ -292,9 +324,10 @@ class Resolvable:
     - Keeps track of what caused the resolvable to enter existence (it's cause)
     """
 
-    UNRESOLVED = 'unresolved'
     ERROR = 'error'
+    LATE = 'late'
```

Comment:
More of a todo: consider if enum would be suitable here.
1) https://docs.python.org/3/library/enum.html
2) https://github.com/mesosphere/mergebot/blob/master/mergebot/mergebot/interfaces.py#L13